src/providers/VoiceFocusProvider/docs/useVoiceFocus.stories.mdx

```diff
@@ -0,0 +1,106 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { useVoiceFocus } from '../';
+
+<Meta title="SDK Hooks/useVoiceFocus" />
+
+# useVoiceFocus
+
+The `useVoiceFocus` hook returns a function transforming a normal device to an Amazon voice focus device, and the support state of voice focus status.
+
+## Return Value
+
+```typescript
+{
+  // Whether the voice focus is supported. The default value is undefined, then changes to true or false.
+  isVoiceFocusSupported: boolean | undefined;
+
+  // A function to transform an audio input device to an Amazon voice focus device
+  addVoiceFocus: (device: Device) => Promise<Device | VoiceFocusTransformDevice>;
+}
+
+```
+
+## Importing
+
+```javascript
+import { useVoiceFocus } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+The hook depends on the `VoiceFocusProvider`. You can use it with `MeetingProvider`.
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  VoiceFocusProvider,
+  useMeetingManager,
+  useVoiceFocus,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <VoiceFocusProvider>
+    <MeetingProvider>
+        <MyChild />
+    </MeetingProvider>
+  </VoiceFocusProvider>
+);
+
+const MyChild = () => {
+  const meetingManager = useMeetingManager();
+  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
+  const [isVoiceFocusEnabled, setIsVoiceFocusEnabled] = useState(false);
+  const [device, setDevice] = useState(meetingManager.selectedAudioInputTransformDevice)
+  const {isVoiceFocusSupported, transformToVoiceFocus} = useVoiceFocus();
+
+  useEffect(() => {
+    meetingManager.subscribeToSelectedAudioInputTransformDevice(setDevice);
+    return (): void => {
```

Comment:
Same as for VF provider, can remove `: void` as we say `jsx`.

Reply:
updated